ts/packages/anchor/src/coder/borsh/event.ts

```diff
@@ -70,4 +83,26 @@ export class BorshEventCoder implements EventCoder {
     const data = layout.decode(logArr.slice(8));
     return { data, name: eventName };
   }
+
+  get idlAddress(): PublicKey {
+    return new PublicKey(this.address);
+  }
```

Comment:
I think it would be better to store `idlAddress` as `PublicKey` and remove this getter.